src/plugin/tune-rpc.ts

```
@@ -14,12 +14,57 @@ function stepsByOrder(lesson: FrameNode) {
     })
 }
 
+function getPaintColor(paint: Paint) {
+  switch (paint.type) {
+    case 'SOLID': {
+      const c = paint.color
+      return { r: c.r * 255, g: c.g * 255, b: c.b * 255 }
+    }
+    default: {
+      console.log(`Warning! Unsupported paint type ${paint.type}`)
+      return { r: 166, g: 166, b: 166 }
+    }
+  }
+}
+
+function getColor(node: SceneNode) {
+  let color = { r: 166, g: 166, b: 166 } // default color
+  switch (node.type) {
+    // TODO: check other cases
```

Ну а зачем оставлять наполовину доделанную задачу, надо доделать, у нас много где используются другие виды "листьев".

Ты даже тайпскрипту тут не угождаешь, раз у тебя SceneNode остается тип, смысл в свитче?
Просто проверь наличие fills/strokes и вынимай оттуда, если они есть. Код получится короче и понятней. Логи эти тоже убирай. Let it fail.

---

Done